src/jsonrpc/miner_jsonrpc_snapshot.erl

```diff
@@ -0,0 +1,39 @@
+-module(miner_jsonrpc_snapshot).
+
+-include("miner_jsonrpc.hrl").
+
+-behavior(miner_jsonrpc_handler).
+
+-define(MAX_LOOKBACK, 25). %% only search the previous 25 blocks
```

Comment:
unused?

Reply:
Copypasta mistake that wasn't cleaned up; removed